R/copy-model-from.R

```diff
@@ -22,6 +22,7 @@
 #'   and ancestry. **Paths must be relative to `.new_model` path.** Note that
 #'   the `.parent_model` will automatically be added to the `based_on` field, so
 #'   no need to include that here.
+#' @param .starred boolean variable to denote significance
```

Comment:
Might just be me, but I'd suggest avoiding "denote significance", particularly in a model context.  I liked @kylebaron's phrasing in gh-474: "star certain models to indicate special interest level".  Since "star" is in the parameter name, perhaps "mark model to indicate special interest level".

Also, what do you think about`.star` rather than `.starred`?  That'd be shorter and seems to capture the intent well enough ("star this model when you copy it").